release.sh

```diff
@@ -109,6 +137,13 @@ case $1 in
             display_help
         fi
     ;;
+    new)
+        if [ -n "${2+x}" ] && [ -z "${3+x}" ]; then
```

Comment:
Doesn't this need to check for `new snapshot`?

```suggestion
        if [ $# -eq 2 ] && [ "$2" == 'snapshot' ]; then
```

Reply:
Right, I should be checking that. I mean it will work as is, but it definitely be more strict :-)

`==` is what I usually use and so far it's not bitten me. I hadn't thought of `$#`, thanks for that! Much better than checking `$3` is empty.